schema.sql

```diff
@@ -1,5 +1,14 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+     animal_name TEXT,
+     DATE_OF_BIRTH DATE,
+     ESCAPE_ATTEMPTS INT,
+     NEUTERED BIT,
+     WEIGHT_KG DECIMAL,
+     SPECIES TEXT,
+     PRIMARY KEY (id)
```

Comment:
- [ ] As it is a requirement for this project, " `Add a column species of type string to your animals table. Modify your schema.sql file.` ", please add a new column called species with a string type (choose the varchar limit that you want), something like so: 

`ALTER TABLE animals ADD species varchar(255);`